html/frontend.js

```diff
@@ -28,11 +28,14 @@ function countChars() {
 }
 
 function decode(string) {
-	return decodeURIComponent(escape(atob(unescape(string).replace(/-/g, "+").replace(/_/g, "/"))))
+        if (string[0] != "*")
```

Comment:
The indentation here is inconsistent with the rest of the file: this is indented with 8 spaces, whereas everything else (including the other lines in your modification) is indented with tabs.

Also, what does JavaScript do if `string` is empty and you try to extract element 0?